keps/sig-apps/2232-suspend-jobs/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: KEP Template
```

Comment:
Please update the title

Reply:
oops, thanks for flagging that. opened https://github.com/kubernetes/enhancements/pull/2423